app/src/main/java/ch/epfl/sdp/appart/Card.java

```diff
@@ -0,0 +1,67 @@
+package ch.epfl.sdp.appart;
+
+import android.media.Image;
+import ch.epfl.sdp.appart.user.User;
+
+public class Card {
+  private final int id;
+  private final User owner;
+  private String title;
+  private String address;
+  private String price;
+  private Image image;
+
+  public Card(int id, User owner) {
+    this.id = id;
+    this.owner = owner;
+
+    //default parameters....
+    this.title = "APPARTAMENT";
+    this.address = "SWISS";
+    this.price = "100fr/month";
```

Comment:
I think the price should be an integer. This way we would be able to handle different currency transparently optionally. What do you think. Also, it avoids to have to check for a specific format when displaying the card.